README.md

```diff
@@ -1,7 +1,26 @@
 Discard Old Build plugin
 ===========================
 
-Jenkins plugin to discard old builds with detailed configuration.
+Pantheon's fork of a Jenkins plugin to discard old builds with detailed configuration.
+
+We have added the option to discard all successful builds, beyond a certain number of kept builds. The existing plugin did not support this functionality out of the box.
+
+Typically we would want to configure our frequently-run jobs using this plugin option set to some numerical limit, e.g. 1000. The most recent 1000 builds will be kept, and then all successful builds beyond 1000 will be deleted. This will leave only the failed, aborted, etc. builds in the range of 1000+. Using it with Jenkin's default option `Discard Old Builds` with a higher limit, say 4000, and the net affect will be that we have unsuccessful builds saved for far longer.
```

Comment:
@nstielau We'd have to play with those figures, could also use a combination of numerical and time based limits. I'll think more about this. With a week or two of time, I feel like I could come up with a good solution - a custom plugin that handles precisely our use case. The reason I said I was "meh" about this, is because this PR is a hacky quick-fix compromise and it doesn't leave me very satisfied quality-wise.